src/cosmosdb-pitr/HISTORY.rst

```diff
@@ -0,0 +1,8 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0
+++++++
+* Initial release.
```

Comment:
Is this editable ? if yes, then let us write proper release notes.

Reply:
Added the release notes. please let me know if this is clear